images/linux/scripts/installers/1804/powershellcore.sh

```diff
@@ -8,11 +8,15 @@
 source $HELPER_SCRIPTS/document.sh
 
 LSB_RELEASE=$(lsb_release -rs)
+echo "Addin the repository [microsoft-ubuntu-bionic-prod] to /etc/apt/sources.list.d/dotnetdev.list"
+sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic main" > /etc/apt/sources.list.d/dotnetdev.list'
```

Comment:
Stop putting the repo ,we already have this repo in the repos.sh script.

Reply:
As far as I remember you've pointed to this code in repos.sh as a potential dangerous,m because it will broke Ubuntu 16 build.